ui/tests/integration/components/ttl-picker2-test.js

```diff
@@ -44,7 +44,7 @@ module('Integration | Component | ttl-picker2', function (hooks) {
       />
     `);
     await click('[data-test-toggle-input="clicktest"]');
```

Comment:
we're calling the onChange method more now because we're matching the state of what's displayed to what the model's value is on init. e.g. we're making sure they both match.